inst/shiny/modules/tlg/tlg_option_table.R

```diff
@@ -0,0 +1,158 @@
+#' Function generating an input widget for TLG option.
+#' @param id      id of the input widget
+#' @param opt_def definition of the option, as specified in the `yaml` file
+#' @param data    data object used for parsing labels, strings, infering placeholder values or
+#'                choices etc.
+tlg_option_table_ui <- function(id, opt_def, data) {
+  ns <- NS(id)
+
+  label <- if (is.null(opt_def$label)) id else opt_def$label
+
+  actionButton(ns("open_table"), label = label)
+}
+
+#' Function generating an input widget server for TLG option.
+#' @param id      id of the input widget
+#' @param opt_def definition of the option, as specified in the `yaml` file
+#' @param data    data object used for parsing labels, strings, infering placeholder values or
+#'                choices etc.
+#' @returns a reactive with the input value
+tlg_option_table_server <- function(id, opt_def, data) {
```

Comment:
Consider adding more details to the docstrings, ex: expected structure / type of `opt_def` and `data`.